openai/api_resources/image.py

```diff
@@ -88,9 +107,9 @@ def _prepare_create_variation(
             api_version=api_version,
             organization=organization,
         )
-        _, api_version = cls._get_api_type_and_version(api_type, api_version)
+        api_type, api_version = cls._get_api_type_and_version(api_type, api_version)
 
-        url = cls._get_url("variations")
+        url = cls._get_url("variations", azure_action=None, api_type=api_type, api_version=api_version)
```

Comment:
Does Azure support this yet? If we pass azure_action=None, will this target the OAI endpoint with azure credentials?

Reply:
we don't, neither this nor edits, I was thinking how to handle this... do you think we raise an exception?

Reply:
I see we've done that in other parts of the library, but not consistently. For example:

https://github.com/openai/openai-python/blob/fe3abd16b582ae784d8a73fd249bcdfebd5752c9/openai/api_resources/edit.py#L21-L24

I think I prefer raising the exception in this case, but @johanste for input.